.rubocop_todo.yml

```diff
@@ -13,7 +13,7 @@ Lint/UnusedBlockArgument:
   Exclude:
     - 'lib/codelog/command/step/version.rb'
 
-# Offense count: 2
+# Offense count: 1
```

Comment:
Can we remove this cop, or fix this offense?

Reply:
I think we need to discuss more about the cops used in this project. Currently we are using a lot of them that I don't really see the necessity.
About this specific cop, for now, I'll let this way until we find a better solution.